src/models/calendar.js

```diff
@@ -78,10 +78,16 @@ export const getDefaultCalendarObject = (props = {}) => Object.assign({}, {
  * @returns {Object}
  */
 export function mapDavCollectionToCalendar(calendar) {
+	let color = calendar.color
```

Comment:
Apparently `color` is not fetched by cdav for some reason and leads to undefined here.